TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift

```diff
@@ -111,21 +111,16 @@ extension FactorFacade {
   class Builder {
     
     private var networkProvider: NetworkProvider!
-    private var keyStorage: KeyStorage!
     private var url: String!
     private var authentication: Authentication!
     private var clearStorageOnReinstall = true
+    private var accessGroup: String?
     
     func setNetworkProvider(_ networkProvider: NetworkProvider) -> Self {
       self.networkProvider = networkProvider
       return self
     }
     
-    func setKeyStorage(_ keyStorage: KeyStorage) -> Self {
```

Comment:
I would prefer to keep this for testing, probably the class creating the `KeyStorage` can add the access group